hack/test-update-storage-objects.sh

```diff
@@ -114,12 +117,16 @@ ${UPDATE_ETCD_OBJECTS_SCRIPT}
 killApiServer
 
 
+
 #######################################################
 # Step 3 : Start a server which supports only the new api version.
 #######################################################
 
 KUBE_API_VERSIONS="${KUBE_NEW_API_VERSION}"
 RUNTIME_CONFIG="api/all=false,api/${KUBE_NEW_API_VERSION}=true"
+
+# This seems to reduce flakiness.
+sleep 1
```

Comment:
Is this still needed even with the wait added in? And if so, maybe you should add it at the end of the killApiServer function, since that case is exercised multiple times?